data-structures-stacks/get-2nd-from-top.js

```diff
@@ -0,0 +1,13 @@
+/* exported get2ndFromTop */
+
+function get2ndFromTop(stack) {
+  let top, secondFromTop;
+
+  if (stack.peek() !== undefined) top = stack.pop();
+  if (stack.peek() !== undefined) secondFromTop = stack.pop();
+
+  if (secondFromTop !== undefined) stack.push(secondFromTop);
+  if (top !== undefined) stack.push(top);
+
```

Comment:
These checks feel redundant, maybe there is a way to restructure so you don't have to check the same things twice 🤔